### PR TITLE
Upgrades: introduce ABtests to Themes nudge

### PR DESCRIPTION
**Pull request by artpi**
Introduces ABtest to notice introduced in #4859 using copy inspired by #4888 

zrzut ekranu 2016-04-20 o 14 35 30

CC @mtias @rralian

### CODE REVIEW

File: client/my-sites/themes/single-site.jsx [hunk 1]
```diff
@@ -156,12 +157,22 @@ var ThemesSingleSite = React.createClass( {
...
+					<UpgradeNudge
+						title={ this.translate( 'Get Custom Design with Premium' ) }
+						message={ this.translate( 'Customize your theme using premium fonts, color palettes, and the CSS editor.' ) }
+						feature="custom-design"
+						event="themes_custom_design"
+					/>
+				) }
+				{ ( abtest( 'themesHeaderNudge' ) === 'themes_premium_theme_more_traffic' ) && (
+					<UpgradeNudge
+						title={ this.translate( 'Blogs with Premium theme get at least 7 times more traffic' ) }
```
Comment (rralian) [inline client/my-sites/themes/single-site.jsx]: style thing, but I'd get rid of "at least" to make it a little punchier.

File: client/my-sites/themes/single-site.jsx [hunk 2]
```diff
@@ -156,12 +157,22 @@ var ThemesSingleSite = React.createClass( {
...
+						message={ this.translate( 'Get unlimited premium themes, more storage and Google Analytics with Business Plan.' ) }
```
Comment (rralian) [inline client/my-sites/themes/single-site.jsx]: please add the Oxford comma (before the "and")

**Comment by rralian** [comment]
Looks reasonable. Can you also write a p2 post announcing this test, give a baseline, and try to determine how long it should be run for?

File: client/lib/abtest/active-tests.js [hunk 3]
```diff
@@ -73,6 +73,15 @@ module.exports = {
 		defaultVariation: 'drake',
 		allowExistingUsers: true,
 	},
+	themesHeaderNudge: {
+		datestamp: '20160409',
```
Comment (rralian) [inline client/lib/abtest/active-tests.js]: don't forget to change the date when this launches

File: client/my-sites/themes/single-site.jsx (hunk 1 above)
Comment (artpi) [inline client/my-sites/themes/single-site.jsx]: Well, our data indicates that's actually 30 times more, but that sounded less believable...

File: client/my-sites/themes/single-site.jsx (hunk 2 above)
Comment (artpi) [inline client/my-sites/themes/single-site.jsx]: done!

File: client/lib/abtest/active-tests.js (hunk 3 above)
Comment (artpi) [inline client/lib/abtest/active-tests.js]: I will!

**Comment by artpi** [comment]
> Looks reasonable. Can you also write a p2 post announcing this test, give a baseline, and try to determine how long it should be run for?

Should I do that before or after merge?

**Comment by rralian** [comment]
> Should I do that before or after merge?

Can be done after.

File: client/my-sites/themes/single-site.jsx (hunk 1 above)
Comment (rralian) [inline client/my-sites/themes/single-site.jsx]: I would change it, it just doesn't flow naturally to me. Also, there's some pluralization issues with the wording. I'd do:
> Blogs with premium themes get over 7 times more traffic
Get premium themes, more storage, and Google Analytics with WordPress.com Business.

**Comment by rralian** [comment]
I think this looks 👍  with the suggested wording changes.

**Pull request merged**